Dockerfile

```diff
@@ -1,12 +1,13 @@
-FROM postgres:9.6
+FROM postgres:10.1
 
-ENV PATH ~/.cargo/bin/:$PATH
 ENV CARGO_HOME /cargo
+ENV PATH $CARGO_HOME/bin:$PATH
```

Comment:
sorry to be picky but do we need this change?

Reply:
Please, picky's fine :)

I changed this because it didn't work without it. Previously the path was being set as `ENV PATH ~/.cargo/bin/:$PATH`, but `~/.cargo/bin` is not where cargo installs itself or its crates.